reasoner/Reasoner.java

```diff
@@ -139,7 +163,9 @@ ResolverRegistry resolverRegistry() {
         return resolverRegistry;
     }
 
-    private ReasonerProducer resolve(Conjunction conjunction) {
-        return new ReasonerProducer(conjunction, resolverRegistry);
+    private Producer<ConceptMap> resolve(Conjunction conjunction) {
+        return Producers.empty();
+        // TODO enable reasoner when ready!
+        // return new ReasonerProducer(conjunction, resolverRegistry);
```

Comment:
the `resolve()` method is integrated now, but returns an empty producer.

Reply:
so we can see the flow that will be required - i didn't want to add the block about `reasonerEnabled`, which fits in the context of this PR, and then just have an empty `if` block that does nothing inside :)